File: Code/App/v2/RTG/ViewModels/RTGExaminationViewModel.cs
using BusinessLogic.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RTG.ViewModels
{
    public class RTGExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public RTGExaminationCommentViewModel RTGComment { get; set; }

    }
}

File: Code/App/separateDB/RTG/Scripts/App/Models/RTGExaminationViewModel.js
function RTGExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.RTGComment = new
        RTGExaminationCommentViewModel(data.RTGComment);
    this.examinationDescriptionRTG = ko.observable();

};